mesh_map/include/mesh_map/util.h

```diff
@@ -58,6 +61,8 @@ typedef lvr2::Normal<float> Normal;
 typedef lvr2::BaseVector<float> Vector;
 
 
+lvr2::MeshBufferPtr extractMeshByName(const aiScene* ascene, std::string name);
```

Comment:
`aiScene` woot, no namespace, instead `assimp` prefix `ai` to their types?

Reply:
Yes, that's how they do it